benchexec/tools/liv.py

```diff
@@ -99,3 +89,58 @@ def get_value_from_output(self, output, identifier):
                         line,
                     )
         return match
+
+
+class Tool(BaseTool2):
+    """
+    Tool info for LIV.
+    """
+
+    def __init__(self):
+        self._cached_version: Optional[str] = None
+        self._original_liv = LivVersionLessThanTwo()
+        self._metaval_liv = MetavalVersionGreaterThanOrEqualTwo()
```

Comment:
I would say it likely is better if there is just one instance that this module delegates to, instead of always instantiating both and then having a case distinction in several method.